src/internal/operators/publishReplay.ts

```diff
@@ -9,6 +9,85 @@ export function publishReplay<T>(bufferSize?: number, windowTime?: number, sched
 export function publishReplay<T, O extends ObservableInput<any>>(bufferSize?: number, windowTime?: number, selector?: (shared: Observable<T>) => O, scheduler?: SchedulerLike): OperatorFunction<T, ObservedValueOf<O>>;
 /* tslint:enable:max-line-length */
 
+/**
+ * Returns a ConnectableObservable, which is a variety of Observable that waits until its connect method is called
+ * before it begins emitting items to those Observers that have subscribed to it. Internally a ReplaySubject is created
+ * to support caching items for replay to Observers that subscribe after emission begins. Cached values can also be delayed
+ * by a given amount of time.
+ *
+ * <span class="informal">Makes a cold Observable hot and allows for caching of source values</span>
+ *
+ * ![](publishReplay.png)
+ *
+ * ## Examples
+ * Make source$ hot by applying publishReplay operator, and cache 1 value in a ReplaySubject
+ * for any observers that subscribe after the source begins emitting.
+ * ```ts
+ * import { of, zip, interval, merge } from "rxjs";
+ * import { map, publishReplay, tap, refCount } from "rxjs/operators";
+ *
+ * const source$ = zip(interval(1000), of(1, 2, 3, 4, 5, 6, 7, 8, 9)).pipe(
```

Comment:
Just use `interval(1000)`, the `zip` and `of` are a little superfluous here, we want to keep the examples as simple as possible.